### PR TITLE
adding bottle framework support

### PR DESCRIPTION
**Pull request by lucasrc**
Adds support for bottle framework, based on the Flask plugin already in the project.

### CODE REVIEW

**Comment by sloria** [comment]
@lucascosta Thanks! This looks great. Looks like there's a couple lint errors; can you fix those please?

**Comment by lucasrc** [comment]
@sloria lint error fixed :+1:

File: .gitignore [hunk 1]
```diff
@@ -33,6 +34,7 @@ nosetests.xml
 .mr.developer.cfg
 .project
 .pydevproject
+.idea
```
Comment (sloria) [inline .gitignore:37]: This is fine, but things like this should really go in a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

**Comment by sloria** [comment]
Looks good to me.

**Pull request merged**